### DISCUSSION TITLE
OLD TESTS

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#737] [Q&A] [answered]
https://github.com/pschatzmann/arduino-audio-tools/blob/e092217341745e1b91ebe250b8026555f260ddc3/sandbox/i2s-file_aac/i2s-file_aac.ino

you mention that above  test failed. So after gong through the code i observe the code is incomplete you  didn't use record function , other then this  their no flow in technique  i guess  ? 

Actually i  try some thing like this yesterday for SBC encoder  and it failed too .  I KNOW ABOVE IS VERY OLD TECHNQUE AND IT DID'NT COMPLE EITHER ,  

I am thinking what cause my yesterday test to fail ?  this is its link
 https://github.com/pschatzmann/arduino-audio-tools/discussions/736

**Comment by pschatzmann** [comment, marked as answer]
Here is a link to a [working, tested example](https://github.com/pschatzmann/arduino-audio-tools/blob/3b5d8aeaa8812a5ba342633dc265e541544c33ac/examples/examples-audiokit/streams-audiokit-webserver_aac/streams-audiokit-webserver_aac.ino)

Anything that does not exist any more is __definitely not supported__!
